packages/neuron-ui/src/contexts/settings.ts

```diff
@@ -0,0 +1,16 @@
+import { createContext } from 'react'
+
+interface IWalletSettings {
+  seeds: string
+  seedsValid: boolean
+  passwordValid: boolean
+}
+
+export const initSettings: IWalletSettings = {
```

Comment:
If possible, I'd like to merge contexts of main components into `wallet context` cause they are paired.

Reply:
we should merge some, or we would have too much wrapper on context in withProviders. it is relative with wallet, although it is setting/creating process info.    I would move it first

Reply:
Yes if we feel there's too many contexts, just refactor and adjust.

(BTW please pay attention to the commit message. Use `feat(neuron-ui)` instead of `feat[neuron-ui]`.